docs/source/release.rst

```diff
@@ -7,33 +7,109 @@ Release Notes
+* :bug:`2010 major` Fix pydocstyle config
+* :support:`2012` Fixed impala start up issues
+* :feature:`2001` Pyspark backend bounded windows
+* :bug:`2006 major` Pinning clickhouse-driver<0.1.2
+* :support:`1999` cache all ops in translate()
+* :feature:`1987` Add more POSTGIS operations
+* :feature:`1969` SQLAlchemy Default precision and scale to decimal types for PostgreSQL and MySQL
+* :support:`1988` Add black step to CI
+* :support:`1962` Json UUID any
+* :bug:`1984 major` Fix CI log for database
+* :feature:`1983` Add support for array operations in PySpark backend
+* :feature:`1978` Implement sort, if_null, null_if and notin for PySpark backend
+* :support:`1982` Add log for database services
+* :feature:`1974` Add support for date/time operations in PySpark backend
+* :feature:`1973` Add support for params, query_schema, and sql in PySpark backend
+* :support:`1972` Fix BigQuery backend fixture so batting and awards_players fixture re…
+* :support:`1971` Disable BigQuery explicitly in all/test_join.py
+* :feature:`1967` Implement join for PySpark backend
+* :feature:`1952` Validate AsOfJoin tolerance and attempt interval unit conversion
+* :support:`1963` Re-formatting all files using pre-commit hook
+* :support:`1961` Disable codecov report upload during CI builds
+* :support:`1960` Developer doc enhancements
+* :feature:`1943` filter for PySpark backend
+* :feature:`1945` window operations for pyspark backend
+* :support:`1958` Missing geospatial ops for OmniSciDB
+* :feature:`1951` Implement IntervalSub for pandas backend
+* :support:`1950` Remove pandas deprecation warnings
+* :support:`1948` Add developer docs to get docker setup
+* :support:`1949` More informative IntegrityError on duplicate columns
+* :feature:`1942` PySpark backend string and column ops
+* :support:`1928` Improve geospatial literals and smoke tests
+* :support:`1925` PostGIS enhancements
+* :bug:`1933 major` Fixes explain operation
+* :feature:`1913` PySpark backend
+* :bug:`1937 major` Fix incorrect assumptions about attached SQLite databases
+* :bug:`1938 major` Upgrade to JDK11
+* :support:`1866` Rename mapd to omniscidb backend
+* :support:`1926` Fix failing BigQuery tests
+* :feature:`1908` DDL support for Spark backend
+* :support:`1917` Added missing null literal op
+* :feature:`1923` Support timezone aware arrow timestamps
+* :bug:`1903 major` `sql` method doesn't work when the query uses LIMIT clause
+* :feature:`1860` Add shapely geometries as input for literals
+* :bug:`1910 major` Fix union implementation
+* :bug:`1912 major` Fix failing com imports on master
+* :feature:`1858` Add geopandas as output for omniscidb
+* :bug:`1901 major` OmniSci/MapD - Fix reduction for bool
+* :feature:`1885` Spark UDFs
+* :feature:`1871` Add support for Postgres UDFs
+* :bug:`1899 major` Pass scope to grouping execution in the pandas backend
+* :support:`1895` Update link to Presto website
+* :support:`1896` Removing linting from windows
```

Comment:
there are no duplicates here?

Reply:
I check it now and I didn't find any duplicates there.